src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationStrictCondition.java

```diff
@@ -7,8 +7,8 @@ void method(Thread foo) { //indent:4 exp:4
         method( //indent:8 exp:8
                 new Thread() { //indent:16 exp:16
                         public void run() { //indent:24 exp:24
-                            } //indent:28 exp:28
+                            } //indent:28 exp:16,20,24 warn
```

Comment:
why we suggesting 16,20 ? all of them less then 24 that is base for this block.

we have the same problems in report
https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/braceAdjustments_2020212202/reports/diff/hibernate-orm/index.html#A1
there are a lot of diffs to suggest values that not good, but previously we suggested single indentation.

Reply:
@romani, I have not tweaked anything related to braces when it is on the **same line**, I have just changed code for braces where left curly starts from a new line.

> why we suggesting 16,20 ? all of them less then 24 that is base for this block.

I tested the same thing with original checkstyle (8.36.2), and with that too, 3 levels of suggested indents were shown.(possibly due to `new` keyword)(see the image below)
In the above case you had mentioned; the indentation suggestion is actually wrong while tested with the original checkstyle because there should be no affect of brace adjustment on rcurly if it is written according to modern style, and thats why the new changes were showing error on it.

![Screenshot from 2020-09-30 22-21-23](https://user-images.githubusercontent.com/48255244/94715879-acc66800-036b-11eb-931f-2afed38419ea.png)

>  there are a lot of diffs to suggest values that not good, but previously we suggested single indentation.

It's only the case when the braces are on new line.
**Why**
Because we have to satisfy both cases (google as well as normal), so **one extra** indent level for google only.